### ISSUE TITLE
Docker build stopped working

### ISSUE THREAD

**Issue by thelicato** [rbsec/sslscan#290]
I tried building a Docker image with the ``make docker`` command and it gave me the following error:
```
0.157 git clone --depth 1 -b `git ls-remote https://github.com/openssl/openssl | grep -Eo '(openssl-3\.0\.[0-9]+)' | sort --version-sort | tail -n 1` https://github.com/openssl/openssl ./openssl && cd ./openssl && touch ../.openssl.is.fresh ; \
0.157 fi
0.158 sort: unrecognized option: version-sort
```

That's because the ``sort`` command does not have the ``--version-sort`` option in ``busybox``. But that's not the main issue; in fact by replacing the ``--version-sort`` option with ``-V`` I encounter the following error:

```
  24 |     COPY --from=builder /builddir/sslscan /sslscan
  25 |     COPY --from=builder /lib/libz.so.1 /lib/libz.so.1
  26 | >>> COPY --from=builder /lib/ld-musl-x86_64.so.1 /lib/ld-musl-x86_64.so.1
  27 |
  28 |     # Drop root privileges.
--------------------
ERROR: failed to solve: failed to compute cache key: failed to calculate checksum of ref 6f97c354-5640-48ed-9948-bbf8c2201752::227xiwgwc0th43drqizw1mjnq: "/lib/ld-musl-x86_64.so.1": not found
```

**Comment by jtesta** [comment]
@thelicato : I've submitted a PR (#291) to fix this issue.  Thanks for reporting!

**Comment by rbsec** [comment]
Fixed in #291 - thanks as always @jtesta
